packages/runtime/lib/start.js

```diff
@@ -100,6 +100,10 @@ async function start (args) {
 
 async function startCommand (args) {
   try {
+    // adds default allow env vars to runtime
+    if (!args.includes('--allow-env')) {
+      args.push('--allow-env', 'PORT,HOSTNAME,DATABASE_URL')
+    }
```

Comment:
This is not the right place for those.

Use https://github.com/platformatic/platformatic/blob/af21dfce22b5085a153afe970f62352062edf23e/packages/runtime/lib/config.js#L240 like https://github.com/platformatic/platformatic/blob/af21dfce22b5085a153afe970f62352062edf23e/packages/db/index.js#L112